requirements/edx/base.txt

```diff
@@ -94,6 +94,7 @@ requests-oauthlib==0.4.1
 scipy==0.14.0
 Shapely==1.2.16
 singledispatch==3.4.0.2
+six>=1.10.0,<2.0.0
```

Comment:
[`six`](https://pythonhosted.org/six/) is a library to help make code compatible with both Python 2 and 3.